src/lib.rs

```diff
@@ -49,6 +49,9 @@ lazy_static! {
 
     /// The docker tag used for the Synapse image we produce.
     static ref PATCHED_IMAGE_DOCKER_TAG: OsString = OsString::from_str("mx-tester/synapse").unwrap();
+
+    /// The name of the docker network to use for mx-tester.
+    static ref MX_TEST_DOCKER_NETWORK: String = "mx-tester".to_string();
```

Comment:
I think that we should rather pass the network name as an env variable, in case some day we need to run more than once instance of `mx-tester` concurrently?

Reply:
You mean something that is given to mx-tester and not the other way around? otherwise there is no easy way to track what network should be being used e.g. someone uses `up` and then later `down` or `up` twice in a row. I agree if we are providing it to mx-tester though

Reply:
Why would you want it to be an environment variable and not something to put into mx-tester.yml?

Reply:
I meant that `mx-tester` should set e.g. `MX_TEST_DOCKER_NETWORK=mx-tester` and pass this to scripts. Is this a problem?

Reply:
> Would it make sense to start using https://crates.io/crates/rs-docker instead of calling "docker" as a process and parsing the result?

It would (not sure how well that is maintained or documented), but it is probably better to move all of that to another patch

Reply:
Good point.